README.md

```diff
@@ -21,3 +20,45 @@ Have a look at the file for details, but in short this is how it works:
 - Create a custom docker image for Drupal and nginx, and push those to a docker registry (typically that of your cloud provider).
 - Install or update our helm chart while passing our custom images as parameters.
 - The helm chart executes the usual drush deployment commands.
+
+## Local development
+
+Install [Lando](https://docs.devwithlando.io/). Add your project's name and other project-specific settings to `.lando.yml` and run `lando start`.
+
+Useful helper commands:
+
+- `lando cre` runs `composer require`
+- `lando cup` runs `composer update`
+- `lando cupdc` runs `composer update drupal/core webflo/drupal-core-require-dev --with-dependencies`
+- `lando dsi` runs `drush si --db-url=mysql://drupal8:drupal8@database:3306/drupal8`. Run `lando dsi demo_umami -y` to install `demo_umami` profile.
+
+Run `lando` for complete list of available Lando commands.
+
+### Testing
+
+Supported testing tools:
+
+#### [PHP_CodeSniffer](https://github.com/squizlabs/PHP_CodeSniffer) with Drupal, DrupalPractice standards
+
+Usage: `lando phpcs <path>`
+
+Examples:
+
+- `lando phpcs web/modules/custom`
+- `lando phpcbf web/modules/custom`
+
+#### [Codeception](https://github.com/Codeception/Codeception)
+
+Usage: `lando codeception <command>`
+
+Create Codeception tests folder by running `lando codeception bootstrap`.
```

Comment:
We should use the same default setup that Guntis has created for the WunderTools repository.

Reply:
`lando codeception run acceptance` and `lando codeception run unit` seem to be working now, `lando codeception run functional` gives an error:

```
Codeception PHP Testing Framework v2.5.4
Powered by PHPUnit 6.5.14 by Sebastian Bergmann and contributors.
Running with seed:

In Drupal.php line 130:

  \Drupal::$container is not initialized yet. \Drupal::setContainer() must be called with a real container.
```

Unfortunately, I'm just in the beginning of becoming a testing guru, so some help is needed here - @guncha25, @floretan :)

Reply:
Thanks, @smmccabe for the 💯`crap` 💯  here: https://github.com/lando/lando/issues/1090#issuecomment-419269001 :)

Reply:
This is probably ok for now and needs project-specific adjustments.